todo/src/components/AddTodo.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { addTodo } from "../actions";
+
+
+ class AddTodo extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            todoText: "",
+        }
+    }
+     handleChanges = ev => {
+        this.setState({ [ev.target.name]: ev.target.value });
+      };
+       handleClick = ev => {
+        ev.preventDefault();
+        console.log("Button clicked");
+        this.props.addTodo(this.state.todoText);
+      };
+     render() {
+        return (
+            <div>
+                <form onSubmit= {ev => {
```

Comment:
might be good to define this as a method, so it's not getting redefined on every re-render